changes/3646.added

```diff
@@ -0,0 +1,2 @@
+Redirect unauthenticated users on either `/api/docs/` or `/graphql/` to login page if `HIDE_RESTRICTED_UI` is True.
```

Comment:
My read of the first list item in #3646 is that with HIDE_RESTRICTED_UI, *all* views should redirect to the login for unauthenticated users. Does that match your understanding?

Reply:
Yr, but those has been implemented before